api/apps/api/src/modules/scenarios/marxan-run/run.handler.ts

```diff
@@ -24,6 +31,23 @@ export class RunHandler {
     private readonly blmDefault: number,
   ) {}
 
+  async runCalibration(scenarioId: string, blmValues: number[]) {
+    const assets = await this.assets.forScenario(scenarioId, 0);
+    assertDefined(assets);
+    const job = await this.calibrationQueue.add(`calibrate-scenario`, {
```

Comment:
typings of queues/bull are wrong. `job` may be undefined

```
if (!job) {
      this.logger.error(
        `Unable to start job PlanningUnitsJob - adding job failed.`,
      );
      return;
    }
```

Reply:
I've added the check :+1: